proto/viam/app/v1/app.proto

```diff
@@ -787,6 +793,22 @@ message RemoveRoleRequest {
 
 message RemoveRoleResponse {}
 
+message AddRoleToAPIKeyRequest {
```

Comment:
we might want to call this AddAuthorizationToAPIKeyRequest because we never add a role without attaching a resource to it (that combination is called an authorization)

Reply:
and same for remove as well

Reply:
Yeah it makes perfect sense, thank you so much for your comment. question: an authorization is a combination of a role and a resource; in this context, what does a resource mean? Is a specific robot a resource that an api key is granted access to?

Reply:
yes! a resource right now is either an org, location, or robot.

Reply:
Thank you so much!!